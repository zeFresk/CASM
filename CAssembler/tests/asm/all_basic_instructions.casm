.start 30
inp 01
out 02
lda 03
ldi 04
sta 05
sti 06
add 07
sub 08
jaz 09
hrs 00
