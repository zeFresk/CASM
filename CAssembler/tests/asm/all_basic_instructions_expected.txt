30
30: 001
31: 102
32: 203
33: 304
34: 405
35: 506
36: 607
37: 708
38: 809
39: 900
